packages/tsconfig-4catalyzer/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "tsconfig-4catalyzer",
```

Comment:
is the name good?

Reply:
Maybe `@4c/tsconfig`, can we add a workspaces base version in this package as well, like just an additional file to cherry pick from take from frontend

Reply:
should I just add `"plugins": [{ "name": "typescript-workspace-plugin" }]`?